web/v1/snowflake/sql-runner/sql/standard/00-setup/StoredProcedures.md

```diff
@@ -0,0 +1,84 @@
+# Stored procedures
```

Comment:
I think we can remove this readme if commit_tables() can be simplified. If not, we can keep the readme but let's just mention `commit_tables()`, rather than explaining all the others (which we don't expect to be used).

Reply:
Ok. What about `mk_transaction`?

Reply:
Do we expect it to be used directly? Seems to me that it's contained within `commit_table()`, no?

Reply:
No, it is not used within `commit_tables`. But it can be used in any upsert e.g. in custom modules.

Reply:
Ok, I'm confused - when do we use it then?

Seems fine to leave it in but I'm confused about why it exists if we already provide a way to commit upserts

Reply:
Sorry if i added confusion, maybe i am the one confused. `mk_transaction` is a way to wrap statements into a transaction. `commit_tables` does this for the commit step of the modules (including some more checks). So far, i indeed agree that `commit_tables` involves transaction logic but i consider them as different. Am i wrong here?

Reply:
Yes I do see the difference between these - I think I phrased my question poorly. When do we need `mk_transaction`?

Reply:
Currently `mk_transaction` is being used in: `98-base-manifest` and `98-manifest-and-truncate` of base and users module respectively and in `02-page-views-join-upsert` of the custom module.

Reply:
Ah ok I see now. Ok cool, there's good reason to keep that too yeah.